src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -1015,29 +1015,62 @@ namespace CoolingPanelSimple {
 		
 		RegisterPlantCompDesignFlow( CoolingPanel( CoolingPanelNum ).WaterInletNode, CoolingPanel( CoolingPanelNum ).WaterVolFlowRateMax );
 
+		bool SizeUAErrorFlag = false;
+		SizeCoolingPanelUA( CoolingPanelNum, SizeUAErrorFlag );
+		if ( SizeUAErrorFlag ) ShowFatalError( "SizeCoolingPanelUA: Program terminated for previous conditions." );
+
+	}
+
+	void
+	SizeCoolingPanelUA(
+					 int const CoolingPanelNum,
+					 bool & SizeUAError
```

Comment:
@RKStrand I think maybe I was unclear on what I was suggesting here, sorry if that was the case. What I was suggesting here was to change the function signature to `bool SizeCoolingPanelUA(int const CoolingPanelNum)` rather than use a reference.